pcsx2/gui/Panels/CpuPanel.cpp

```diff
@@ -251,8 +251,8 @@ void Panels::CpuPanelEE::ApplyConfigToGui( AppConfig& configToApply, int flags )
 	m_panel_RecEE->Enable(!configToApply.EnablePresets);
 	m_panel_RecIOP->Enable(!configToApply.EnablePresets);
 
-	m_check_EECacheEnable ->SetValue(recOps.EnableEECache);
-	m_check_EECacheEnable->Enable(!configToApply.EnablePresets);
+	m_check_EECacheEnable->SetValue(recOps.EnableEECache);
+	m_check_EECacheEnable->Enable(!configToApply.EnablePresets && !m_panel_RecEE->GetSelection());
```

Comment:
I still prefer a comment - If I don't have PCSX2 open and I'm not conveniently hovering over the tooltip then it's not immediately obvious to me.

The other is fine (you don't have to change it), though I still prefer the other way.